.github/workflows/check.yml

```diff
@@ -1,13 +1,14 @@
 name: 'Terraform check'
 on:
-  push:
   pull_request:
 permissions:
   id-token: write
   contents: write
 jobs:
   check:
-    uses: nationalarchives/tdr-github-actions/.github/workflows/terraform_check.yml@main
+    uses: nationalarchives/dr2-github-actions/.github/workflows/terraform_check.yml@main
```

Comment:
See comment above about the repo referenced here.

Reply:
The check job is a tricky one because you can't have an input parameter telling you which project it is so it's difficult to make it generic which is why I've just done this specific one. It's running `terraform validate` and `terraform fmt` so it applies to all projects. It's a bit rubbish but I can't think of a better solution yet.